src/secret_base.c

```diff
@@ -964,7 +964,7 @@ static void HandleRegistryMenuInput(u8 taskId)
 
     data = gTasks[taskId].data;
     input = ListMenu_ProcessInput(data[5]);
-    ListMenuGetScrollAndRow(data[5], &data[2], &data[1]);
+    ListMenuGetScrollAndRow((u8)data[5], (u16 *)&data[2], (u16 *)&data[1]);
```

Comment:
Explicit cast from a signed integer type to a narrower unsigned integer type is not required.